### PR TITLE
Add telomerase activity prediction module authors

### PR DESCRIPTION
**Pull request by jaclyn-taroni**
### Purpose

Addresses #115 - adds authors of the telomerase activity prediction module to the author list in `content/metadata.yaml`.

### Directions for reviewers. Tell potential reviewers what kind of feedback you are soliciting.

@syzheng and @NNoureen can you double check the information I've added here and make sure you agree with this change? Thanks!

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -328,6 +338,18 @@ authors:
...
+    initials: SZ
+    email: zhengs3@uthscsa.edu
+    twitter: zhengsiyuan
+    orcid: 0000-0002-1031-9424
+    contributions:
+      - Formal analysis
+      - Writing - Original draft
+      - Supervision
+    affiliations:
+      - Greehey Children's Cancer Research Institute, UT Health San Antonio
```
Comment (syzheng) [inline content/metadata.yaml:368]: looks good to me! Thanks

Comment (NNoureen) [reply content/metadata.yaml:368]: >   - github: NNoureen
    name: Nighat Noureen
    initials: NN
    orcid: 0000-0001-7495-8201
    email: noureen@uthscsa.edu
    contributions:
      - Formal analysis
      - Writing - Original draft
    affiliations: 
      - Greehey Children's Cancer Research Institute, UT Health San Antonio


It's okay.. Thanks

**Review by cbethell — APPROVED** [review]
🚀

**Review by jashapiro — APPROVED** [review]
🚀

**Pull request merged**